Makefile

```diff
@@ -136,6 +138,10 @@ kubebuilder:
 	mv /tmp/kubebuilder_2.3.2_${OS}_${ARCH} ${PWD}/.bin/kubebuilder
 	export PATH=${PATH}:${PWD}/.bin/kubebuilder/bin
 
+.bin/ory: Makefile
+	curl https://raw.githubusercontent.com/ory/meta/master/install.sh | bash -s -- -b .bin ory v0.1.43
```

Comment:
Maybe it would be better to extract the version and set as a parameter/env in the makefile?

Reply:
@Demonsthere happy to do that if it adds something. My concern is that this value is used only once, so extracting this number into a "constant" only adds complexity without clear benefits. My current approach is to keep it as simple and straightforward. "What version of `.bin/ory` does this Makefile use? Let's check where it installs it... Aha, v0.1.43!"